src/pages/academy/components/video-banner/carousel/_custom-carousel.tsx

```diff
@@ -25,63 +25,83 @@ import { useRecursiveTimeout } from 'components/hooks/use-recursive-timeout'
 import { Header } from 'components/elements'
 import { Flex } from 'components/containers'
 
-export const PrevButton = ({ color, enabled, is_reviews, onClick, style }) => (
+const StyledDiv = styled.div<{ style?: string; onClick?: () => void }>``
+
+export type ButtonsProps = {
```

Comment:
```suggestion
export type PrevButtonProps = {
```
I suggest using a non-universal type for each component props so that TS warns us if we pass wrong props to a component 🙂

Reply:
Done, @maryia-binary 
Thanks for explanation and good point 🙏